src/toil/cwl/cwltoil.py

```diff
@@ -3743,6 +3766,15 @@ def main(args: Optional[List[str]] = None, stdout: TextIO = sys.stdout) -> int:
                     )
                     raise
 
+                # Attempt to prepull the containers
+                if not options.no_prepull:
+                    if not options.enable_ext:
+                        # The CWL utils parser does not support cwltool extensions and will crash if encountered, so don't prepull if extensions are enabled
+                        # See https://github.com/common-workflow-language/cwl-utils/issues/309
+                        try_prepull(uri, runtime_context, toil.config.batchSystem)
+                    else:
+                        logger.debug("Not prepulling containers as cwltool extensions are not supported.")
```

Comment:
I think this is better as a debug message, but it could be a warning if need be.